### DISCUSSION TITLE
Smart blinds

### DISCUSSION THREAD

**Discussion by TrDA-hab** [arendst/Tasmota#10847] [Show and tell]
## 0. How to use it.
- [More information about Shutters](https://tasmota.github.io/docs/Blinds-and-Shutters)   
- [More information on commands for Shutters](https://tasmota.github.io/docs/Commands/#shutters)   
- Now it is highly recommended to use TMC 2208 instead of А4988 (!).   
- [Step by step instructions](https://github.com/arendst/Tasmota/discussions/11731).   

## 1. MID roller blinds.
- 3D models and BOM you can find here (optional):   
https://www.thingiverse.com/thing:4163340   

![416](https://raw.githubusercontent.com/TrDA-hab/Projects/master/Smart%20blinds/MID%20dlinds/416-2.jpg)
![](https://raw.githubusercontent.com/TrDA-hab/Projects/master/Smart%20blinds/MID%20dlinds/20200301_183806.jpg)
![Board-416](https://raw.githubusercontent.com/TrDA-hab/Projects/master/Smart%20blinds/MID%20dlinds/Board-416.jpg)
![](https://raw.githubusercontent.com/TrDA-hab/Projects/master/Smart%20blinds/MID%20dlinds/Box%20v4-4.png)
![](https://raw.githubusercontent.com/TrDA-hab/Projects/master/Smart%20blinds/MID%20dlinds/Box%20v5-1.png)
![](https://raw.githubusercontent.com/TrDA-hab/Projects/master/Smart%20blinds/MID%20dlinds/Box-416.jpg)
![](https://raw.githubusercontent.com/TrDA-hab/Projects/master/Smart%20blinds/MID%20dlinds/t10.png)
![](https://raw.githubusercontent.com/TrDA-hab/Projects/master/Smart%20blinds/MID%20dlinds/20200409_232319.jpg)
![](https://raw.githubusercontent.com/TrDA-hab/Projects/master/Smart%20blinds/MID%20dlinds/20200409_233041.jpg)
![](https://raw.githubusercontent.com/TrDA-hab/Projects/master/Smart%20blinds/MID%20dlinds/20200416_211911.jpg)

## 2. BIG roller blinds.
- 3D models and BOM you can find here (optional):   
https://www.thingiverse.com/thing:4301158   

![412](https://raw.githubusercontent.com/TrDA-hab/Projects/master/Smart%20blinds/BIG%20blinds%20-2/Blinds-412-2.jpg)
![412](https://raw.githubusercontent.com/TrDA-hab/Projects/master/Smart%20blinds/BIG%20blinds%20-2/Board-412.jpg)  
![412](https://raw.githubusercontent.com/TrDA-hab/Projects/master/Smart%20blinds/BIG%20blinds%20-2/002.PNG)
![412](https://raw.githubusercontent.com/TrDA-hab/Projects/master/Smart%20blinds/BIG%20blinds%20-2/Roller-gear%20v.2%20v36.png)
![412](https://raw.githubusercontent.com/TrDA-hab/Projects/master/Smart%20blinds/BIG%20blinds%20-2/Roller-gear%20v.2%20v35.jpg)
![412](https://raw.githubusercontent.com/TrDA-hab/Projects/master/Smart%20blinds/BIG%20blinds%20-2/Roller-gear%20v.2%20v37.png)
![412](https://raw.githubusercontent.com/TrDA-hab/Projects/master/Smart%20blinds/BIG%20blinds%20-2/20200416_163055.jpg)
![412](https://raw.githubusercontent.com/TrDA-hab/Projects/master/Smart%20blinds/BIG%20blinds%20-2/20200416_164141.jpg)
![412](https://raw.githubusercontent.com/TrDA-hab/Projects/master/Smart%20blinds/BIG%20blinds%20-2/20200416_164537.jpg)
![412](https://raw.githubusercontent.com/TrDA-hab/Projects/master/Smart%20blinds/BIG%20blinds%20-2/20200416_170800.jpg)

## 2.1. BIG roller blinds + INA219.
![432](https://raw.githubusercontent.com/TrDA-hab/Projects/master/Smart%20blinds/INA219/A4988-INA219%20v1.jpg)
![432](https://raw.githubusercontent.com/TrDA-hab/Projects/master/Smart%20blinds/INA219/A4988-INA219%20v2.jpg)
![](https://raw.githubusercontent.com/TrDA-hab/Projects/master/Smart%20blinds/INA219/20200620_173756.jpg)
![](https://raw.githubusercontent.com/TrDA-hab/Projects/master/Smart%20blinds/INA219/Roller-gear%20v.3%20v20.png)

## 3. x2 Blinds.
- 3D models and BOM you can find here (optional):   
https://www.thingiverse.com/thing:4163505   
- [Video](https://youtu.be/TFUa2v-jqgw)   

![422](https://raw.githubusercontent.com/TrDA-hab/Projects/master/Smart%20blinds/Blind%20x2/422-1.jpg)
![422](https://raw.githubusercontent.com/TrDA-hab/Projects/master/Smart%20blinds/Blind%20x2/422-2.jpg)
![422](https://raw.githubusercontent.com/TrDA-hab/Projects/master/Smart%20blinds/Blind%20x2/Board%20-%20422.jpg)
![422](https://raw.githubusercontent.com/TrDA-hab/Projects/master/Smart%20blinds/Blind%20x2/blinds-x2-NEMA-17%20v23-1.jpg)
![422](https://raw.githubusercontent.com/TrDA-hab/Projects/master/Smart%20blinds/Blind%20x2/blinds-x2-NEMA-17%20v23-2.png)
![422](https://raw.githubusercontent.com/TrDA-hab/Projects/master/Smart%20blinds/Blind%20x2/20200307_174831.jpg)
![422](https://raw.githubusercontent.com/TrDA-hab/Projects/master/Smart%20blinds/Blind%20x2/20201213_172915.jpg)
![422](https://raw.githubusercontent.com/TrDA-hab/Projects/master/Smart%20blinds/Blind%20x2/20201213_172828.jpg)
![422](https://raw.githubusercontent.com/TrDA-hab/Projects/master/Smart%20blinds/Blind%20x2/20191130_192502.jpg)
![422](https://raw.githubusercontent.com/TrDA-hab/Projects/master/Smart%20blinds/Blind%20x2/20200411_203935.jpg)
![422](https://raw.githubusercontent.com/TrDA-hab/Projects/master/Smart%20blinds/BIG%20blinds%20-2/007-2.jpg)
![422](https://raw.githubusercontent.com/TrDA-hab/Projects/master/Smart%20blinds/Blind%20x2/20200412_153343.jpg)

## 4. Venetian blinds (jalousie).
- [How to use DC motors](https://github.com/arendst/Tasmota/discussions/10387).
- [video #1](https://youtu.be/2r1aYv723qo).     
- [video #2](https://youtu.be/QtgAn3ByDkc).   

**Best regards,    
TrDA.**

**Comment by willpuckett** [comment]
Great post!

**Comment by soo246** [comment]
3. x2 Blinds.
This part

Can I get the full firmware??

**Reply by barbudor** [reply]
What 'full firmware'?
All tasmota pre-built firmwares are available on github or http:/ota.tasmota.com
You need standard 'tasmota.bin' for blind and shutters.
For more questions, please join the Discord server.
